absDir: "/home/shapelim/dataset"
targetLoc: "HD_B1"
date: "2019-04-16_15-35-46" #"2019-04-16_15-35-46"
voxelSize: 0.15
targetLidar: "both" # "lidar0", "lidar1", or "both"
initIdx: 19000
stopIdx: -1 # If not use, please set the value as "-1"
interval: 2
frequency: 1
savePath: "/home/shapelim/GN_B1.pcd"

erasor: 
    max_range: 5.0  # 20 / 15 also good     10/5 For Korea Univ!!
    num_rings: 3    # 35   # 0911_{3, 4} 는 20이었음
    num_sectors: 20 # 108
    min_h: -0.1
    max_h: 2.0
    th_ground_retrieval_h: 0.2 # <- 0.4   height [m]
    th_bin_max_h: 0.2 # [m]
    scan_ratio_threshold: 0.2 # percentage: originally, 0.3 yet 0.3 seems like it is quite aggressive!
    minimum_num_pts: 6
    rejection_ratio: 0
    sac_distThr: 0.15
    gf_distThr: 0.15 # originally, 0.125
    gf_iter: 3
    gf_num_lpr: 10
    gf_th_seeds_height: 0.2
    version: 2
    voxel_size: 0.075






